print/src/main.rs

```diff
@@ -92,6 +87,7 @@ fn main() -> anyhow::Result<()> {
                 Err(defmt_decoder::DecodeError::UnexpectedEof) => break,
                 Err(defmt_decoder::DecodeError::Malformed) => {
                     log::error!("failed to decode defmt data: {:x?}", frames);
+                    #[allow(clippy::try_err)]
```

Comment:
I think I agree with Clippy here – this would be clearer if it used `return`.

Reply:
Done!